### ISSUE TITLE
Stilts fails to recognize control flow expression when there's a new line.

### ISSUE THREAD

**Issue by jquesada2016** [Atrociously/stilts#8]
Currently, my HTML is being formatted like this because the HTML formatter feels like it lol:

```html
<input class="input {% kind %} {% class %}" type="{% type_ %}" name="{% name %}" placeholder="{% placeholder %}" {% if
  *required %} required="" {% end %}>
```

Under this formatting, stilts fails to pick up the `if` expression because there's a new line between `{% if` and the end of the expression start, `*required %}`.

If I remove the newline, then all is well again. Perhaps we can filter out all whitespace when parsing the expression?

**Comment by Atrociously** [comment]
I just released a new version of the language parser which should allow for this. Personally if it were not auto formatting I wouldn't format my code like this, however I can see the value in using a consistent auto formatter!

**Comment by Atrociously** [comment]
I'm gonna close this issue as solved with the release of 766ad733175a0d8344534b3e6562e47be56ad9a3
